src/Engine.php

```diff
@@ -112,6 +112,13 @@ public function execute(int $threads): bool
 
     private function runInitialTestSuite(): void
     {
+        if ($this->config->shouldSkipInitialTestSuite()) {
+            $this->consoleOutput->logSkippingInitialTestSuite();
+            $this->assertCodeCoverageExists(null, $this->config->getTestFramework());
```

Comment:
Probably, it makes sense to create 2 methods and don't use a nullable argument with `if/else`. This would be more readable.

Reply:
Is this ok: https://github.com/infection/infection/pull/1042/files#diff-5a8536ad09be922aea0b4725411fea45R180